giraffe/src/utils/__mocks__/getTextMetrics.ts

```diff
@@ -0,0 +1,4 @@
+export const getTextMetrics = (...args) => ({
+  width: Math.floor(args[1].length * 1.2),
+  height: 12,
```

Comment:
maybe add a comment to explain magic numbers ?

Reply:
Sure thing, will do that.